src/backend/distributed/commands/table.c

```diff
@@ -1758,18 +1772,29 @@ PreprocessAlterTableSchemaStmt(Node *node, const char *queryString,
 	{
 		return NIL;
 	}
+
+	QualifyTreeNode((Node *) stmt);
```

Comment:
as we discussed, would be nice to (a) make this change as a separate commit in case we might want to backport this to earlier versions (b) have a simple test for sequences as well